torch/_inductor/autotune_process.py

```diff
@@ -34,58 +34,45 @@ class Pong:
 
 @dataclasses.dataclass
 class TuningProcess:
-    process: Optional[BaseProcess] = None
-    request_queue: Optional["Queue[Any]"] = None
-    response_queue: Optional["Queue[Any]"] = None
+    process: Optional["subprocess.Popen[bytes]"] = None
 
     @staticmethod
-    def process_main(
-        request_queue: "Queue[Any]",
-        response_queue: "Queue[Any]",
-    ) -> None:
-        print("enter child process main")
-        while True:
-            obj = request_queue.get()
+    def process_main() -> None:
+        """
+        Work loop for the benchmarking subprocess.
+        """
+        if DEBUG:
+            print("Entering TuningProcess child main", file=sys.stderr)
+
+        def reply(obj):
```

Comment:
Any reason we can not reuse put method below?

```
   def put(self, obj: Any) -> None:
        """
        Push a work item to the child process.
        """
        assert self.process is not None
        assert self.process.stdin is not None
        pickle.dump(obj, self.process.stdin)
        self.process.stdin.flush()
```

Reply:
NVM, they are different. Might be good to change this as a method together with put/get methods below since they are very similar